orders/OrderFormNotion.ts

```diff
@@ -68,4 +78,14 @@ export default class OrderForm {
             console.warn(`Order form checker: An error occurred when checking for order form updates.`, e);
         }
     }
+
+    private completedSushiOrder(orderObject: PageObjectResponse) {
```

Comment:
nit: add return type
```suggestion
    private completedSushiOrder(orderObject: PageObjectResponse): boolean {
```